src/common/harvest/harvest.js

```diff
@@ -263,7 +262,7 @@ function createAccumulator () {
   var accumulator = {}
   var hasData = false
   return function (key, val) {
-    if (val !== null && val !== undefined && val.length) {
+    if (val !== null && val !== undefined && val.toString()?.length) {
```

Comment:
Does this handle empty objects appropriately? (all objects.toString() will just create a string of `[object Object]` so just double checking this is intended

Reply:
`mapOwn` takes care of empty objects.